### PR TITLE
Setup and mobile first

### PR DESCRIPTION
**Pull request by ghreza-crypto**
Portfolio project first section ( toolbar and headline section)
First review:
-Readme file updated.
1-logo wrapped with a link tag.
2-article tag changed to p tag.
3-social tags surrounded with (ul-li-a) tags.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav id="phone">
+            <div class="logo">My Logo</div>
```
Comment (RiyaBulia12) [inline index.html]: It is recommended to wrap your logo around a link (**a tag**). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 👍🏽 

- [ ] Kindly add the `<a>` tag inside `<div>` for the logo  `<div class="logo"><a href="#" ... >My Logo </a> </div>` 😃 
- [ ] Please make sure that the changes you make do not affect the UI. If it does can you please change the CSS as well? 💯

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            <img src="images/humbarger.svg" alt="humbarger" class="humbarger">
+        </nav>
+    </header>
+    <main>
+
+        <section id="bg">
+            <section id="bio">
+                <h1 class="title">I’m Mariane Glad to see you!</h1>
+                <article class="about">I’m a software developer! I can help you build a product , feature or website
```
Comment (RiyaBulia12) [inline index.html]: As per the requirement, `I'm a software... ` should be wrapped inside `<p></p>` tag. 
- [ ] Can you please change `<article>` tag  to `<p>` ? 👍🏽 
- [ ] Kindly test the UI and please fix the styles if its broken. 🟢 💯

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                </article>
+                <div class="social-title">LET’S CONNECT</div>
+                <div class="social">
+                    <img src="images/social/twitter.svg" alt="twitter">
+                    <img src="images/social/linkedin.svg" alt="linkedin">
+                    <img src="images/social/m.svg" alt="m">
+                    <img src="images/social/github.svg" alt="github">
+                    <img src="images/social/hand.svg" alt="hand">
+
+                </div>
```
Comment (RiyaBulia12) [inline index.html]: As per the requirement, the list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
- [ ] How about changing the `<div>` to `<ul>`and wrap the `` inside `<li><a href="..."><a></li>` as per the requirement? 💯 😉 
```
<ul class="social">
   <li>
        <a href="...">
            <img src="images/social/twitter.svg" alt="twitter">
         </a>
   </li>
   ...
</ul>
```

- [ ] Please test the UI and make the changes in the CSS if required. 🟢 💯

**Review by RiyaBulia12 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi  @ghreza-crypto  👋, 

Your project looks fantastic. Congratulations! 🎉 🎉 🎉  
There are a few issues that you still need to fix but you are almost there!

## Highlight 🎉
- No linter errors :heavy_check_mark:
- Correct GitHub flow :heavy_check_mark:
- Professional Readme  ✔️ 
- Achieved most of the requirements ✔️ 
- UI matches the Figma Template and looks professional ✔️ 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._ 🙂 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                <a href="#">My Logo</a>
+            </div>
+            <img src="images/humbarger.svg" alt="humbarger" class="humbarger">
+        </nav>
+    </header>
+    <main>
+
+        <section id="bg">
+            <section id="bio">
+                <h1 class="title">I’m Mariane Glad to see you!</h1>
```
Comment (amejid) [inline index.html:26]: - [ ] [OPTIONAL] I think it would be great if you put your name here instead of `Mariane` to make the portfolio tailored to you. 😎️

**Review by amejid — APPROVED** [review]
Hi @ghreza-crypto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**